command/server/config.go

```diff
@@ -58,6 +59,9 @@ type Headers struct {
 // minimum block generation time in seconds
 const defaultBlockTime uint64 = 2
 
+// default value for all pubSub buffers
+const defaultBufferSize uint64 = 512
+
```

Comment:
This naming is not descriptive.

You are altering the size of the outbound **and** validation message queues in this PR, but you're noting this as changing the size of a single ambiguous buffer.

Please consider naming this something like `defaultOutputBufferSize`, and expand on the explanation above it